api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p><code>first_location_data[-1]</code> could be replaced with <code>most_recent_data_point</code> in several places to make use of that variable you just defined in the above cell.</p><p> </p><p>Also, using <code>datetime.strptime(date[:10], '%Y-%m-%d')</code> seems kind of hacky to me. Could accomplish the same thing in a more explicit way with:</p><p> </p><p><code>TIMESTAMP_FMT = '%Y-%m-%dT%H:%M:%S.%fZ'  # good to define once and re-use in many places</code></p><p><code>datetime.strptime(date, TIMESTAMP_FMT)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>okay </p>